### ISSUE TITLE
Version command prints "unknown"

### ISSUE THREAD

**Issue by sed-i** [canonical/pebble#212]
```bash
$ juju exec --unit prometheus/0 -- pebble version
client  unknown
server  -

$ juju exec --unit prometheus/0 -- PEBBLE_SOCKET=/charm/containers/prometheus/pebble.socket pebble version
client  unknown
server  unknown
```

**Comment by benhoyt** [comment]
Yeah, we're going to start creating proper tagged releases soon (and use those in Juju, among other places). Leaving this open to track.

**Comment by benhoyt** [comment]
Note that this is in place now (Pebble has tagged released). However, we're stil using an older, non-tagged release in Juju, but we're going to update that soon. We should check, though: I'm pretty sure that because Juju is building Pebble separately from source, it probably won't embed the version number correctly even if we start using the version tag in Juju's `go.mod`. Just something to be careful about.

**Comment by benhoyt** [comment]
And as a workaround, you can figure out the version of Pebble by going to Juju's go.mod of the Juju version you're using, and looking at the commit hash for the `canonical/pebble` line.

**Comment by benhoyt** [comment]
This will be fixed when #225 is fixed, so closing as duplicate of that one.
